lib/config/presets/internal/workarounds.ts

```diff
@@ -21,7 +21,7 @@ export const presets: Record<string, Preset> = {
       {
         matchDatasources: ['maven', 'sbt-package'],
         matchPackagePrefixes: ['commons-'],
-        allowedVersions: '!/^200\\d{5}(\\.\\d+)?/',
+        allowedVersions: '!/^200/',
```

Comment:
i think this is too loose, please make more tight.

this will block `200.0.0` and I'm sure we don't want that.

Reply:
In the middle of suggesting alternatives, I noticed that the old Regex should work, but it doesn't.

Only if I copy paste exactly the same settings, the PR for 20040616 is closed.

I can see the workaround in the log, but it doesn't seem to work if it's not defined manually.